doc/source/dev/contributor/benchmarking.rst

```diff
@@ -104,17 +104,17 @@ To run benchmarks from a particular benchmark module, such as
 ``optimize_linprog.py``, simply append the filename without the
 extension::
 
-   python runtests.py --bench optimize_linprog
+   python dev.py --bench optimize_linprog
 
 To run a benchmark defined in a class, such as ``KleeMinty`` from
 ``optimize_linprog.py``::
 
-   python runtests.py --bench optimize_linprog.KleeMinty
+   python dev.py --bench optimize_linprog.KleeMinty
 
 To compare benchmark results between the active branch and another, such
 as ``main``::
 
-   python runtests.py --bench-compare main optimize_linprog.KleeMinty
+   python dev.py --bench-compare main optimize_linprog.KleeMinty
```

Comment:
If I try this on the command line on latest `main`:

`python dev.py --bench-compare main optimize_linprog.KleeMinty`


![image](https://user-images.githubusercontent.com/7903078/179425918-2a9a703a-cc14-45f0-bb50-b995eea7257f.png)

Something like this seems to work a little better:

`python dev.py bench --compare main`

You may need the `-t` to narrow the benchmark scope beyond that, I didn't check but easy enough to verify locally.